scripts/Interface/ui/drill/test/DrillSampleTest.py

```diff
@@ -0,0 +1,71 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+
+from Interface.ui.drill.model.DrillSample import DrillSample
+
+
+class DrillSampleTest(unittest.TestCase):
+
+    def setUp(self):
+        self.sample = DrillSample()
+
+    def test_init(self):
+        self.assertDictEqual(self.sample._parameters, {})
+
+    def test_setParameters(self):
```

Comment:
The only issue is that old/new is binary. In theory we might have many versions of rundex format, so you should rather organize separate tests for v1, v2, etc.